tests/clv/test_utils.py

```diff
@@ -146,80 +133,47 @@ def fitted_gg(test_summary_data) -> GammaGammaModel:
 
 
 class TestCustomerLifetimeValue:
+    @pytest.mark.parametrize(
+        "t, discount_rate, expected_change",
+        [
+            (1, 0, 1),
+            (1, 1, 0.5),
+            (2, 0, 2),
+            (2, 1, 0.75),
+        ],
+    )
     def test_customer_lifetime_value_bg_with_known_values(
-        self, test_summary_data, fitted_bg
+        self, fitted_bg, t, discount_rate, expected_change
     ):
-        # Test adapted from
+        # Test borrowed from
         # https://github.com/CamDavidsonPilon/lifetimes/blob/aae339c5437ec31717309ba0ec394427e19753c4/tests/test_utils.py#L527
 
-        t = test_summary_data.head()
+        # time=1, discount_rate=0 means the clv will be the same as the predicted
+        # time=1, discount_rate=1 means the clv will halve over a period
+        # time=2, discount_rate=0 means the clv will be twice the initial
+        # time=2, discount_rate=1 means the clv will be twice the initial
```

Comment:
Is this commented code intentional?

Reply:
Yes; previously this test had a lot of redundant code because it wasn't parametrized. These comments explain the expected behavior of the various parameterizations.